web/html/src/branding/css/susemanager/components/buttons.suma.less

```diff
@@ -202,6 +202,7 @@
 .input-group-btn .btn {
   border-width: 1px;
   padding: 6px 12px;
+  height: 35px;
```

Comment:
This should be `@input-height` not a hardcoded value unless I misunderstand the intent here?